purchase_create_invoice_restrict/security/purchase_security.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="group_create_bill_allowed" model="res.groups">
+        <field name="name">Bill Creation Allowed</field>
```

Comment:
```suggestion
        <field name="name">Allow Bill Creation</field>
```
Not a blocking issue, but following suit to how other technical groups are named.